monitor/main.go

```diff
@@ -114,9 +114,13 @@ func runNodeMonitor() {
 	defer server1_0.Close() // Stop accepting new v1.0 connections
 	log.Infof("Serving cilium node monitor v1.0 API at unix://%s", defaults.MonitorSockPath1_0)
 
+	server1_2 := buildServerOrExit(defaults.MonitorSockPath1_2)
+	defer server1_2.Close() // Stop accepting new v1.0 connections
```

Comment:
typo, s/1.0/1.2/

Reply:
yeah... I tried making this a loop, but it ended up being even more complicated. I'll fix the typoes though